embedded-selenium/src/main/java/com/playtika/test/selenium/EmbeddedSeleniumBootstrapConfiguration.java

```diff
@@ -222,4 +228,8 @@ private boolean isValidIpAddress(String ipAddress) {
         }
         return InetAddresses.isInetAddress(ipAddress);
     }
+
+    private boolean isNotBlank(String str) {
```

Comment:
i wonder if there some StringUtils in classpath which already implement this logic

Reply:
There is one in:
 org.testcontainers.shaded.org.apache.commons.lang.StringUtils.isNotBlank

but it doesnt look like a stable api to me. I would have inlined it, but I prefer private methods.